dataclients/kubernetes/definitions/routegroupvalidator.go

```diff
@@ -1,11 +1,19 @@
 package definitions
 
 import (
+	"errors"
+	"fmt"
+
 	"github.com/zalando/skipper/eskip"
 )
 
 type RouteGroupValidator struct{}
 
+var (
+	errMultipleFilters    = errors.New("single filter expected")
+	errMultiplePredicates = errors.New("single predicate expected")
```

Comment:
Since we use this for both no filter and multiple filters I think it should be named like `errSingleFilterExpected`